training/watch.md

```diff
@@ -0,0 +1,29 @@
+---
+---
+
+# 3. When do you watch spam?
+Adding websites and keywords to be watched by SmokeDetector is a big part of being a privileged
+user on Charcoal. Unless a post is ***obvious*** spam, it can sometimes be a bit overwhelming to
+keep up with new software terms, libraries and projects (nuget, chocolatey, mustache, etc.) and
+know what is legit and what is not. We recommend doing a bit of research before you add a watch.
+
+ - **Search** for the URL or keyword on [Stack Exchange](https://stackexchange.com/)
+   - Use `url:example.com` in the search bar to find all questions and answers that have that website
+     in them. Check how many hits come up and if it's only the spam and 2-3 other posts it will
+     typically be OK to add it.
+   - Keywords can be searched just like normal text, but multiple keywords should be enclosed in
+     quotation marks. Example: `"key word"` and `"more words to search for"`. In this way it will
+     only show results for the actual text you search for. Stack Exchange will otherwise show results
+     for each searched word and this will typically return too many hits.
+ - **Check** [open pull requests](https://github.com/Charcoal-SE/SmokeDetector/pulls) on SmokeDetector
```

Comment:
Might be worth mentioning here to also check the history of chat messages after the report to see if someone else has already submitted a watch? But I'm not really sure this is all that critical. If there is a clash/conflict, you'll get a warning. The only harm is a little bit of noise and a little bit of unnecessary load on the SD/MS instance, neither of which seem like a good reason to overburden human volunteers.

Reply:
Is this not what the PRs show? You can quickly see if anything is added to the queue. Otherwise maybe people could run a test !!/test the thing they want to add first. Dunno.